daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/BG.tdml

```diff
@@ -25,25 +25,25 @@
     description="Text number properties">
```

Comment:
We are making many changes to existins tests in the test-ibm1 project. Do we have compatability concerns to worry about?

Reply:
I don't think these are actually IBM tests. They're just incorrectly in the test-ibm1 project (looks like they got moved in commit 406c470a5b in 2013). All the AA-BG tests were either added by Tresys or inherited from the original Daffodil codebase. So it's fine to change these (and they were definitely broken).

Reply:
75 or so tests did originate with IBM, which donated them to the DFDL Workgroup and the Daffodil effort 7 years back. 

The AA-ZZ (aka the "two letter tests") originated with the original Daffodil NCSA code base before anybody including myself started working with this code. 

So calling this bunch of tests "ibm1" is really misleading. Nevertheless keeping them distinct from the more organized and methodically created daffodil-test suite makes sense.